### ISSUE TITLE
The architecture of Retina and batchnorm moving variance and moving mean

### ISSUE THREAD

**Issue by phamquandung** [facebookresearch/Detectron#995]
Dear all, 

I hope that everyone stays safe!

Since this is the first time I tried on detectron, I am familiar with this framework. I would like to have 2 questions. Please help me.

1. After downloading the pre-trained model of RetinaNet baselines  R-50-FPN (https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md), I extracted the weight layer by layer and I got the data like below figures. However, I do not know where these layer in Retina Net. How can I know the architecture of RetinaNet? 

2. In these data, I realized that there are bn_s and bn_b, I guess that they are batch norm. Are they variance and mean?

Thank you for your time and consideration.

<img width="458" alt="Screen Shot 2020-05-09 at 10 40 44 PM" src="https://user-images.githubusercontent.com/28943896/81475886-bf6bc380-9249-11ea-927f-489395cb6300.png">
<img width="456" alt="Screen Shot 2020-05-09 at 10 41 12 PM" src="https://user-images.githubusercontent.com/28943896/81475889-c561a480-9249-11ea-9e64-6fcc6cb9ee43.png">
